src/components/Preferences.react.js

```diff
@@ -41,14 +41,14 @@ var Preferences = React.createClass({
       vmSettings = (
         <div>
           <div className="title">VM Settings</div>
-          <div className="option">
+          <label className="option">
```

Comment:
Yes, it's possible using "label for=...", but it'll be necessary to assign the IDs for input. Is it ok for you?